src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,93 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  items: string[];
```

Comment:
The `items` prop is not typically part of a pagination component's responsibilities. Consider removing it from the `Pagination` component and handling item rendering in the parent component to adhere to separation of concerns.